docker/package/tezos_baking_wizard.py

```diff
@@ -568,79 +656,108 @@ def systemctl_start_action(self, service):
             + ".service"
         )
 
+    def run_setup(self):
 
-def run_setup(cfg):
+        print("Tezos Baking Wizard")
+        print()
+        print(
+            "Welcome, this wizard will help you to set up the infrastructure",
+            "to interact with the Tezos blockchain.",
+        )
+        print(
+            "In order to run a baking instance, you'll need the following Tezos binaries:\n",
+            "tezos-client, tezos-node, tezos-baker-<proto>, tezos-endorser-<proto>.\n",
+            "If you have installed tezos-baking, these binaries are already installed.",
+        )
+        print("All commands within the service are run under the 'tezos' user.")
+        print()
+        print(
+            "To access help and possible options for each question, type in 'help' or '?'."
+        )
+        print("Type in 'exit' to quit.")
+        print()
 
-    if cfg is not None:
-        setup = Setup(cfg)
+        self.query_step(network_query)
+        self.query_step(service_mode_query)
+        self.query_step(systemd_mode_query)
```

Comment:
why not move this to the very end?

After all we'll need to `start` the services either way and this will give a chance to the user to make some check (if he wants to) before actually enabling them.